src/main/java/org/sagebionetworks/bridge/exporter/scheduler/BridgeExporterScheduler.java

```diff
@@ -89,6 +92,12 @@ public void schedule(String schedulerName) throws IOException {
                 String yesterdaysDateString = LocalDate.now(timeZone).minusDays(1).toString();
                 String tag = "[scheduler=" + schedulerName + ";date=" + yesterdaysDateString + "]";
                 requestNode.put("date", yesterdaysDateString);
+                // also, put startDateTime and endDateTime
+                DateTime startDateTime = DateTime.now(timeZone).minusDays(1).withHourOfDay(0).withMinuteOfHour(0).withSecondOfMinute(0).withMillisOfSecond(0);
+                DateTime endDateTime = DateTime.now(timeZone).withHourOfDay(0).withMinuteOfHour(0).withSecondOfMinute(0).withMillisOfSecond(0).minusMillis(1);
```

Comment:
For daily, we use end of the previous day. For hourly, we use start of the current hour. This is inconsistent. We should make these consistent.